Sources/HaCWebsiteLib/views/base.swift

```diff
@@ -0,0 +1,24 @@
+import HaCTML
+
+private let defaultTitle = "Cambridge's student tech society | Hackers at Cambridge"
+
+private func stylesheet(forUrl: String) -> Node {
```

Comment:
Nit: Would be nice to have parameter label `url` (or `urlString`) here instead of `forUrl`

Reply:
I thought that these were supposed to read like sentences? "Give me the `stylesheet for url XXX`.

Reply:
Ideally yeah but if the function name and the label don't give enough context for what the parameter should be then breaking the grammar is alright. e.g. `stylesheet(forUrl urlString: String)`